strongbox-security/strongbox-user-management/src/test/java/org/carlspring/strongbox/users/service/UserServiceTest.java

```diff
@@ -243,10 +248,52 @@ public void testThatUserNameIsUnique()
     @Test
     public void testPrivilegesProcessingForAccessModel()
     {
+        String testUserName = "test-user";
+
+        UserDto userAdd = new UserDto();
+        userAdd.setEnabled(true);
+        userAdd.setUsername(testUserName);
+        userAdd.setPassword("test-password");
+        userAdd.setSecurityTokenKey("before");
+
+        UserAccessModelDto userAccessModelDto = new UserAccessModelDto();
```

Comment:
Cosmetic: Could you please make the code below a nit more readable by organizing it into logical blocks surrounded by empty lines like this for example:
```
Foo foo1 = new Foo();
foo1.setBlah(1);
foo1.setOtherBlah(2);
foo1.doBar();

myService.save(foo1);

logger.info("Saved foo1.");

Foo foo2 = new Foo();
foo2.setOtherBlah(3);
foo2.doBar()

myService.save(foo2);

logger.info("Saved foo2.");
```

When you have one endless block of code that looks like this it's really not pleasant to have to review, as you might loose track and miss something like the `foo1.hide()` below, for example:
```
Foo foo1 = new Foo();
foo1.setBlah(1);
foo1.setOtherBlah(2);
foo1.doBar();
myService.save(foo1);
logger.info("Saved foo1.");
Foo foo2 = new Foo();
foo2.setOtherBlah(3);
foo2.doBar()
myService.save(foo2);
logger.info("Saved foo2.");
foo2.iWasHere(new Date());
foo2.doIt(5, 3, 1);
foo1.hide();
foo2.doThis();
```

Reply:
done!